docs/topics/api/hero.rst

```diff
@@ -41,8 +41,8 @@ small number of shelves this endpoint is not paginated.
     :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
     :>json array results: The array containing the results for this query.
     :>json object results[].gradient: The background colors used for the gradient.
-    :>json string results[].gradient.start: The starting color for gradient - typically top or left.
-    :>json string results[].gradient.end: The ending color for gradient - typically bottom or right.
+    :>json string results[].gradient.start: The starting color name for gradient - typically top or left. The name is from the `photon color variables <https://github.com/FirefoxUX/photon-colors/blob/master/photon-colors.scss>`_.
+    :>json string results[].gradient.end: The ending color name for gradient - typically bottom or right. The name is from the `photon color variables <https://github.com/FirefoxUX/photon-colors/blob/master/photon-colors.scss>`_.
```

Comment:
There's something weird happening with the RST -> html rendering meaning there's an extra `<p>` inserted in the middle of the gradient.end line after the `:`.  If either one of the links is broken/removed then there isn't an extra `<p>` - it only occurs when both of them are in the same list of parameters/response.

![Screenshot 2019-08-19 18 00 17](https://user-images.githubusercontent.com/768592/63345984-ebf24f00-c34b-11e9-8af5-f99be4c49afb.png)